src/elastic-search/mappings/datasetFieldMapping.ts

```diff
@@ -0,0 +1,67 @@
+import { MappingObject } from "../interfaces/mappingInterface";
+
+export const datasetMappings: MappingObject = {
+  description: {
+    type: "text",
+    analyzer: "autocomplete",
+    search_analyzer: "autocomplete_search",
+    fields: {
+      keyword: {
+        type: "keyword",
+        ignore_above: 256,
```

Comment:
Do we really limit the search to 256 characters?

Reply:
Type keyword means that the text will be stored as a whole which can only be searched by the entire text. So, unless an user want to search by writing an article, character limits under 256 should be more than enough